src/cable/payload.cr

```diff
@@ -1,82 +1,118 @@
 module Cable
-  class Payload
```

Comment:
ohhh now I see what you're saying about the payload on the connection, maybe I misunderstood the issue 🤦 

this is cool for sure, even better with the performance boost 👏